.github/workflows/podPublish.yml

```diff
@@ -1,12 +1,14 @@
 name: Pod-Publish
 on:
   push:
+    branches:
+      - main
+      - main_0.1
     tags:
-      - 0.[0-9]+.[0-9]+ # Run automatically on new tags matching a 0.N.M pattern
+      - [0-9]+.[0-9]+.* # Run automatically on new tags matching a 0.N.M pattern
```

Comment:
Wouldn't this be like the below?
0.<number>.<number>[optional_suffix]

0.[0-9]+.[0-9]+*

Reply:
+ right next to * I think it is invalid regex. what I am not too confident about is that GitHub action tag filter seems to be slightly different. like I can't use \d or anything https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet. also I purpose change from 0. to [0-9]+

Reply:
Yeah I think Rafael's _should_ work since GitHub's language is not technically regex, but I wouldn't trust it without trying it. Looks super strange, being used to regex...

Reply:
what I am more nervous about is that I don't have a good way to test this locally. unless you all have better ideas?

Reply:
One way I can think of in order to test this without further side effects is to create a dummy workflow that applies only to a test branch and apply the tags on that test branch.

Reply:
verified on my test branch